WD-Firmware/comrv/api_inc/comrv_defines.h

```diff
@@ -27,9 +27,90 @@
 /**
 * include files
 */
+#include "comrv_config.h"
 
 /**
 * definitions
 */
+// TODO: ronen - use psp defines
+#define D_COMRV_INLINE     __attribute__((inline))
```

Comment:
Consider: if the use must be "inline" then there is "always inline" attribute.

Reply:
done in f224b36